### DISCUSSION TITLE
Removing listener in gridview.vue [May 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2244] [Discord Threads]
> 🙋‍♂️ **Mr.Raccoon**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### sorry, you will have to be more specific what is the problem you are facing or solving  
#  
> 🙋‍♂️ **jiub**  
#### When we trying to add one more event listener to GridView.vue and switch between tables - the gridview component didn't destroy, and addEventListener triggers two times.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978285629794295858)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
